test/trigger_test.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+# NOTE: this script will execute from the repository root when called on jenkins
+
+VAULT_TOKEN=$1
+INPUTS_JSON=$2
+WDL_FILE=$3
+DEPENDENCIES_JSON=$4
+WD=$(pwd)
+echo $(ls)
+echo ${WD}
+
+echo "Rendering templates"
+bash ${WD}/test/config/render-ctmpls.sh "dev" "${VAULT_TOKEN}"
+
+echo "Setting Cromwell environmental variables"
+CROMWELL_SECRETS="${WD}/test/cromwell-secrets.json"
+CROMWELL_USER=$(cat "${CROMWELL_SECRETS}" | jq -r .cromwell_user)
+CROMWELL_PASSWORD=$(cat "${CROMWELL_SECRETS}" | jq -r .cromwell_password)
+
+OPTIONS_FILE="https://raw.githubusercontent.com/HumanCellAtlas/skylab/master/test/options.json"
+CROMWELL_URL="https://cromwell.mint-dev.broadinstitute.org"
```

Comment:
you may want to make this an optional input into the script

Reply:
Thought about it; can do it later if/when we want this to change, I think. For now I wanted to keep the number of input parameters down to keep more state defined in the repo as opposed to in the jenkins job. 

Thoughts?

Reply:
yup that makes a  lot of sense